CMakeLists.txt

```diff
@@ -267,10 +261,10 @@ if(AMENT_ENABLE_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
-  file(
-    COPY "${PROJECT_NAME}"
-    DESTINATION "${CMAKE_CURRENT_BINARY_DIR}"
-  )
+file(
+  COPY "${PROJECT_NAME}"
```

Comment:
I looked into it and I think what's going on is the best way to do it for now. I added a comment to the CMake code explaining what's going on (AFAIK): 20288a8